terraform/modules/sops-credentials/kms.tf

```diff
@@ -0,0 +1,71 @@
+resource "aws_kms_key" "sops_credentials_primary" {
+  description = "Key for encrypting/decrypting secrets in the digitalmarketplace-credentials repo using Mozilla SOPS"
+  policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "AWS": "arn:aws:iam::${data.aws_caller_identity.current.account_id}:root"
+      },
+      "Action": "kms:*",
+      "Resource": "*"
+    },
+    {
+      "Effect": "Deny",
+      "NotPrincipal": {
+        "AWS": "arn:aws:iam::${data.aws_caller_identity.current.account_id}:root"
+      },
+      "Action": [
+        "kms:EnableKeyRotation",
+        "kms:ScheduleKeyDeletion"
+      ],
+      "Resource": "*"
+    }
+  ]
+}
+EOF
+}
+
+resource "aws_kms_alias" "sops_credentials_primary" {
+  name = "alias/sops-credentials"
+  target_key_id = "${aws_kms_key.sops_credentials_primary.key_id}"
+}
+
+resource "aws_kms_key" "sops_credentials_secondary" {
```

Comment:
If the Ireland region becomes inaccessible we can still decrypt the credentials with the other master key.